shallow_backup/__main__.py

```diff
@@ -64,6 +66,10 @@ def cli(show, all, dotfiles, configs, packages, fonts, old_path, new_path, remot
 	# User entered a new path, so update the config
 	if new_path:
 		abs_path = os.path.abspath(new_path)
+
+		if new_dir_is_valid(abs_path):
```

Comment:
```suggestion
		if not new_dir_is_valid(abs_path):
```

Same issue as in the other spot?

Reply:
oop missed that

Reply:
ff1233adcbd4a3b0720889f3cca03ee2f2ff4e8e